### ISSUE TITLE
documentation generator shouldnt resolve jsr imports

### ISSUE THREAD

**Issue by crowlKats** [jsr-io/jsr#396]
This breaks linking to other jsr packages. We would need to somehow reverse lookup export maps of all dependencies, which is not ideal. 
The documentaiton generator should rather not have resolved jsr dependecy URLs so we can properly link to dependencies and their symbols used.
Related to #384

**Comment by lucacasonato** [comment]
Unblocked by https://github.com/jsr-io/jsr/pull/398

**Comment by kellnerd** [comment]
I assume this is the issue which I am observing on [this documentation page](https://jsr.io/@kellnerd/listenbrainz/doc/parser/musicbrainz/~/parseMusicBrainzRelease), where the link to a type `Release` from another JSR package is broken:

https://jsr.io/@kellnerd/musicbrainz/~/Release 404s, it should be https://jsr.io/@kellnerd/musicbrainz/doc/~/Release

Please let me know in case that is another problem and I can create a separate issue.

**Comment by lucacasonato** [comment]
This issue is fixed, but I pulled @kellnerd's issue into #824
